include/nuttx/sensors/sensor.h

```diff
@@ -648,13 +657,14 @@ extern "C"
  *   "upper half" Sensor device and registers that device so that can be used
  *   by application code.
  *
- *   We will register the chararter device by node name format based on the
+ *   You can register the chararter device by node name format based on the
  *   type of sensor. Multiple types of the same type are distinguished by
- *   numbers. eg: accel0, accel1
+ *   numbers. eg: accel0, accel1. This type of sensor must be less than
+ *   SENSOR_TYPE_COUNT. This API corresponds to the sensor_unregister.
  *
  * Input Parameters:
  *   dev   - A pointer to an instance of lower half sensor driver. This
- *           instance is bound to the sensor driver and must persist as long
+ *           instance is bound to the sensor driver and must persists as long
```

Comment:
```suggestion
 *           instance is bound to the sensor driver and must persist as long
```
it was fine as it was